### ISSUE TITLE
Validation set results

### ISSUE THREAD

**Issue by wafaAlghallabi** [hellopipu/PromptMR#4]
Hello,

Thank you very much for your effort and congratulations for this amazing achievement. 

I have two questions if you allow me. 
        1) Regrading the validation set, the challenge submission leaderboard is closed now. May I ask if is there a way to reproduce 
            the results you reported in the paper (I did the training already)?
        2) May I know the system you used for the training please?

Thank you very much for your time.

**Comment by hellopipu** [comment]
1) The CMRxRecon results reported in the paper were trained on 100 cases and evaluated on 20 cases split from the training set (120 cases). The released code has been further optimized, so it should yeild better results than those reported in the paper. For evaluating on the official validation set, the organizers have recently [released](https://www.synapse.org/#!Synapse:syn51471091/datasets/) the fully-sampled data, allowing for local evaluation.

2) please refer to the [installation page](https://github.com/hellopipu/PromptMR/blob/main/INSTALL.md); we trained on two A100 GPUs.
